src/tools/cleaner.js

```diff
@@ -88,15 +88,15 @@ const finalizeBundle = pipe(
 );
 
 // Takes a raw app and bundle and composes a bank of {{key}}->val
-const createBundleBank = (appRaw, event = {}) => {
+const createBundleBank = (appRaw, event = {}, flattenOptions) => {
```

Comment:
As a general style thing, I think we should put any parameters that have defaults at the end of the signature. If I wanted to call this and use the deafult `event`, i'd need to write `createBundleBank(appRaw, undefined, flattenOptions)` which feels cumbersome.

I get that it's used in a pretty specific spot, so maybe it's fine to leave.

Reply:
I generally agree with you @xavdid. I've been in Java land where that pattern was _everywhere_. `flattenOptions` here could have a default as well, but the default is down in `flattenPath` for `options`.

I think `event` has a default in the chance that we don't get it from the integration request. I wouldn't image we'd ever call this function without passing `event`.

We could also just make this a general `options` and have a default as well. I do think that `options` should generally come last.

All that being said, since it _is_ specific, I just moved option down. We should always use these flatten options when creating the bundle bank.